addon/components/mediasuite-radio-buttons-boolean.js

```diff
@@ -1,30 +1,29 @@
 import Ember from 'ember'
-const { computed } = Ember
+const { computed, get } = Ember
 
 export default Ember.Component.extend({
   name: '',
-  value: false,
-  options: computed('name', 'value', 'disabled', function () {
+  computedValue: computed('value', 'options', function () {
+    const value = get(this, 'value')
+    return value == null ? null : get(this, 'options').find(option => option.value === value)
```

Comment:
I'm actually not sure what you'd consider best practices in this regard?  But inclined to leave.  As you've pointed out, it's mainly just internal plumbing to use the mediasuite-radio-buttons component correctly.